package.json

```diff
@@ -14,8 +14,7 @@
     "prettier": "pretty-quick --staged",
     "start": "node ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config ./config/webpack/webpack.dev.js --host 0.0.0.0 --disableHostCheck=true",
     "start-on-server": "node --require dotenv/config ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config ./config/webpack/webpack.dev.js --host 0.0.0.0 --port 80",
-    "snyk": "snyk test",
```

Comment:
This seems to require credentials that I can't find or renew, so I have removed it.